### PR TITLE
made some changes

### PR DESCRIPTION
**Pull request by vigdisha**
the given href for women hack for non profit was outdated

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -302,7 +302,7 @@ <h3>Outspoken Women</h3>
 					<h3>Write/Speak/Code</h3>
 					<p>Visibility and leadership for women & non-binary coders through thought leadership, conference speaking, open source & more.</p>
 				</a>
-				<a class="col-sm-6 col-md-4 p-4" href="http://www.womenhackfornonprofits.com" target="_blank" rel="noopener">
+				<a class="col-sm-6 col-md-4 p-4" href="http://echesters.co.uk/WHFNP-Website/" target="_blank" rel="noopener">
```
[the remark this answers is not in the document]
Comment (vigdisha) [reply index.html:305]: yes, checked.this gives all the required information about the organization

[the remark this answers is not in the document]
Comment (jonatoni) [reply index.html:305]: Did they publish it somewhere on their official channels? I didn't find it anywhere on their Twitter profile for example

[the remark this answers is not in the document]
Comment (vigdisha) [reply index.html:305]: well they have been pretty inactive on twitter ..maybe its better to add the github account

[the remark this answers is not in the document]
Comment (jonatoni) [reply index.html:305]: yeah, can you please replace the link and I can approve it?

[the remark this answers is not in the document]
Comment (vigdisha) [reply index.html]: done.

**Review by Roshanjossey — APPROVED** [review]

**Pull request merged**